testsuite/MDAnalysisTests/analysis/test_msd.py

```diff
@@ -22,6 +22,7 @@
 #
 
 
+from MDAnalysis.analysis.msd import EinsteinMSD as MSD
```

Comment:
I was wondering why this was working and then I realised that when writing that block_import test I had forgotten to take into account that tidynamics isn't imported until you actually run the class (my bad).

I guess technically you don't even need the import inside of `test_notidynamics`. You could also move the test below the `scope=module` fixtures if you think that'd be cleaner.